corehq/ex-submodules/casexml/apps/case/models.py

```diff
@@ -229,6 +229,10 @@ def closed_transactions(self):
         return filter(lambda action: action.action_type == const.CASE_ACTION_CLOSE, self.actions)
 
     @property
+    def transactions(self):
```

Comment:
it looks like you could potentially just workaround this by adding `actions` to the sql model instead of doing it this way